examples/tts/conf/de/fastpitch_align_de.yaml

```diff
@@ -0,0 +1,236 @@
+
```

Comment:
nit: since we are planning to support both 22050 Hz and 44100 Hz, it would be nice to change the config name to something like, `fastpitch_align_22050.yaml`, to distinguish the one withheld for 44100 Hz.